content/reference/admin/private_locations/configuration/ec2/index.md

```diff
@@ -56,9 +56,22 @@ control-plane {
       # profile-name = ""
       # IAM Instance profile (optional)
       # iam-instance-profile = ""
-      # Custom tags
+      # Custom tags (optional)
       tags {
```

Comment:
Remove? deprecated?

Reply:
Neither,
It is the tags that will apply to all resources we know

Reply:
No. It's the config option that we used to have prior to this change that introduced per resource type tags.
It should be deprecated and removed from the documentation.

Reply:
For avoiding breaking change and deprecating, it is the current implemention.
* `tags` (existing config key) was designed to apply all labels to resources we manage (in AWS, we thought that we managed only instance but we learn better) so now, it will apply all labels to all resources we know of (instance, volume, network interface)
* `tags-for` is new and it's designed to let customer specify labels for specific resource type

Reply:
OK